book/src/user/shielded-scan.md

```diff
@@ -9,28 +9,82 @@ Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundati
+Sapling diversifiable/extended full viewing keys strings start with `zxviews` as described in [ZIP-32](https://zips.z.cash/zip-0032#sapling-extended-full-viewing-keys).
+
+For example, to scan for transactions with the [public ZECpages viewing key](https://zecpages.com/z/all) use:
+
+```toml
+[shielded_scan.sapling_keys_to_scan]
+"zxviews1q0duytgcqqqqpqre26wkl45gvwwwd706xw608hucmvfalr759ejwf7qshjf5r9aa7323zulvz6plhttp5mltqcgs9t039cx2d09mgq05ts63n8u35hyv6h9nc9ctqqtue2u7cer2mqegunuulq2luhq3ywjcz35yyljewa4mgkgjzyfwh6fr6jd0dzd44ghk0nxdv2hnv4j5nxfwv24rwdmgllhe0p8568sgqt9ckt02v2kxf5ahtql6s0ltjpkckw8gtymxtxuu9gcr0swvz" = 1
+```
 
 ## Running Sapling Scanning
 
-Launch Zebra and wait for 12-24 hours.
+Launch Zebra and wait. Zebra needs to be synced up to at least the Sapling activation height to start scanning. If you have an already synced-up chain, the scanner will start looking for transactions for the provided keys from the start.
+
+```bash
+zebrad
+```
+
+You should see log messages in the output every 10000 blocks scanned, similar to:
+
+```
+2023-12-16T12:14:41.526740Z  INFO zebra_scan::storage::db: Last scanned height for key number 0 is 435000, resuming at 435001
+2023-12-16T12:14:41.526745Z  INFO zebra_scan::storage::db: loaded Zebra scanner cache
+...
+2023-12-16T12:15:19.063796Z  INFO {zebrad="39830b0" net="Main"}: zebra_scan::scan: Scanning the blockchain for key 0, started at block 435001, now at block 440000, current tip 2330550
+...
+```
+
+The Zebra scanner will resume the task if your Zebra instance went down for any reason. In a new start, Zebra will display:
+
+```
+Last scanned height for key number 0 is 1798000, resuming at 1798001
+```
+
+## Querying Raw Sapling Scanning Results
+
+Install `ldb`:
+
+```bash
+sudo apt install rocksdb-tools
+```
+
+Run `ldb` with the scanner database:
+
+```bash
+ldb --db="$HOME/.cache/zebra/private-scan/v1/mainnet" --secondary_path= --column_family=sapling_tx_ids --hex scan
+```
+
+Some of the output will be markers the scanner uses to keep track of progress, however, some of them will be transactions found.
+
+To lean more about how to filter the database please refer to [RocksDB Administration and Data Access Tool](https://github.com/facebook/rocksdb/wiki/Administration-and-Data-Access-Tool)
 
-## Expert: Querying Raw Sapling Scanning Results
+Querying results this way requires a certain level of expertise.
 
-TODO: Copy these instructions and examples here:
-- https://github.com/ZcashFoundation/zebra/issues/8046#issuecomment-1844772654
+## Quering results with the `scanning-results-reader` utility.
 
-Database paths are different on Linux, macOS, and Windows.
+Another easier way to read the results of found transactions and more is to use the utility tool named `scanning-results-reader`. Please read the [tool description](https://github.com/ZcashFoundation/zebra/tree/main/zebra-utils#scanning-results-reader) to learn how to use it. Instructions are straightforward.
```

Comment:
Many people try instructions in the order that they are written, so I'd suggest putting this paragraph at the start of the querying section.